serialization/items/contentItems/Athlete/6VYohiIqHUW2QvtbYnTOAw.contentItem.yaml

```diff
@@ -8,10 +8,10 @@ fields:
     value: false
     type: Boolean
   dateOfBirth:
-    value: null
```

Comment:
The next time we run the `npm run pull:items` command, those `null` values will come back.

Please test pulling items without running the `removeMediaFromContentItems.ts` script by altering the script in `package.json`. The `null` values might be added by the `js-yaml` module used to read and write back the YAML files. I think the null values style is configurable for the `.dump` function.

If the `null` values are caused by js-yaml, please adjust the dump call parameters to get the desired output.

If the null values are from the ch-one-cli, please adjust the `serialization\scripts\removeMediaFromContentItems.ts` script to remove those `null` values after rewriting the YAML files.

Thanks!

Reply:
The problem is twofold. In the YAML specification, an empty node value should convert to NULL. That said, the Content Hub ONE CLI should support this. I will create a bug report so that Sitecore can fix it.
https://yaml.org/spec/1.2.2/#72-empty-nodes

On the other hand, the Content Hub ONE CLI leaves the empty nodes just empty. It's the js-yaml that converts the empty node value to null, causing a problem when pushing the content items back into a Content Hub ONE instance.

The js-yaml doesn't have a parameter to adjust the dump for this specific use case. For now, I've created a workaround that removes the null values from the parameters. This generates the same output as the Content Hub CLI, solving the issue.